components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +244,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key: string]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+        };
+        type ChipsInfoKey = keyof typeof chipsInfo;
```

Comment:
We could define the type for the `ChipsInfoKey` type beforehand, above in the types definition part of the component.

```typescript
type ChipsInfoKey = 'none' | 'info' | 'warning' | 'error'
```

and assign that to the status in the item definition:

```
type ItemModel = {
    id: string;
    title: string;
    description: string;
    configUrl: string;
    infoUrl: string;
    status: ChipsInfoKey;
}
```

that way we can use it as is without casting it onto the `item.status`

```suggestion
        const chipsInfo: { [key in ChipsInfoKey]: number } = {
            info: 0,
            warning: 0,
            error: 0,
        };
```

Reply:
ok, I like more this approach. Thanks